MultiplayerMod/Multiplayer/State/MultiplayerGame.cs

```diff
@@ -4,14 +4,24 @@ namespace MultiplayerMod.Multiplayer.State;
 
 public static class MultiplayerGame {
 
+    public static IPlayer Player { get; set; } = null!;
     public static MultiplayerRole Role { get; set; }
-    public static bool WorldSpawned { get; set; }
     public static MultiplayerState State { get; set; } = new();
-    public static MultiplayerObjects Objects { get; set; } = new();
+    public static MultiplayerObjects Objects { get; private set; } = new();
+
+    public static PlayerState CurrentPlayerState {
```

Comment:
Maybe it's ok to add something like 
```csharp
    public PlayerState Current => Players[Container.Get<IMultiplayerClient>().Player];
```
into the state itself.

Reply:
Done